experiments/main.py

```diff
@@ -43,14 +43,14 @@
                     type=str,
                     required=True,
                     help="The name of the config generator to evaluate")
-args = parser.parse_args()
+args, other_args = parser.parse_known_args()
```

Comment:
Why use `parse_known_args` here? From this main.py file's perspective, you will be using the configCommand subclass that understands all of the additional options.   
Is this future proofing if you create new ConfigCommand subclasses that dont all overlap?

Reply:
I'm not using configCommand subclass in main.py.  Main is using the ArgumentParser defined right above this line which only knows about a couple options that are specific to this script.

I do agree that using configCommand or something similar in main.py would be the cleaner more robust option. If this was production code I would have done that. I was just trying to cobble together something that allowed me to expose both old and new configuration options via the command-line.

Reply:
cool. thanks for explaining.